src/bika/lims/browser/reports/__init__.py

```diff
@@ -257,11 +257,11 @@ def __call__(self):
         else:
             module = "bika.lims.browser.reports.%s" % report_id
         try:
-            exec ("from %s import Report" % module)
+            Report = getattr(__import__(module), "Report")
```

Comment:
from what I can tell, this is still potentially dangerous because `self.request["report_module"]` can be any module. should we restrict this to trusted modules (in a separate pull request)?